config/gflownet/gflownet.yaml

```diff
@@ -26,7 +24,7 @@ optimizer:
   # Train to sample ratio
   train_to_sample_ratio: 1
   # Number of training iterations
-  n_train_steps: 5000
+  n_train_steps: 2
```

Comment:
Is this intentional?

Reply:
Yep, because I'm still running some tests